Makefile

```
@@ -1,43 +1,37 @@
 status ?= onetime
-version ?= 1.14.6
-deploytool ?= operator
-globalnet ?= false
 build_debug ?= false
 
-TARGETS := $(shell ls scripts | grep -v e2e)
-SCRIPTS_DIR ?= /opt/shipyard/scripts
+ifneq (,$(DAPPER_HOST_ARCH))
 
-.dapper:
-	@echo Downloading dapper
-	@curl -sL https://releases.rancher.com/dapper/latest/dapper-`uname -s`-`uname -m` > .dapper.tmp
-	@@chmod +x .dapper.tmp
-	@./.dapper.tmp -v
-	@mv .dapper.tmp .dapper
+# Running in Dapper
 
-shell:
-	./.dapper -m bind -s
+include $(SHIPYARD_DIR)/Makefile.inc
 
-cleanup: .dapper
-	./.dapper -m bind $(SCRIPTS_DIR)/cleanup.sh
+TARGETS := $(shell ls -p scripts | grep -v -e /)
 
 clusters: build package
-	./.dapper -m bind $(SCRIPTS_DIR)/clusters.sh --k8s_version $(version) --globalnet $(globalnet)
-
-deploy: clusters
-	DAPPER_ENV="OPERATOR_IMAGE" ./.dapper -m bind $(SCRIPTS_DIR)/deploy.sh --globalnet $(globalnet) --deploytool $(deploytool)
 
 e2e: deploy
-	./.dapper -m bind ./scripts/kind-e2e/e2e.sh --status $(status) --deploytool $(deploytool)
+	./scripts/kind-e2e/e2e.sh --status $(status) --deploytool $(deploytool)
 
-$(TARGETS): .dapper vendor/modules.txt
-	./.dapper -m bind $@ --build_debug $(build_debug)
+$(TARGETS): vendor/modules.txt
+	./scripts/$@ --status $(status) --k8s_version $(k8s_version) --deploytool $(deploytool) --globalnet $(globalnet) --build_debug $(build_debug)
```

```suggestion
	./scripts/$@ --build_debug $(build_debug)
```

No need for the flags, only build_debug

---

Done, thanks!